easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.16-goolf-1.7.20.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'Ghostscript'
+version = '9.16'
+
+homepage = 'http://ghostscript.com'
+description = """Ghostscript is a versatile processor for PostScript data with the ability to render PostScript to
+ different targets. It used to be part of the cups printing stack, but is no longer used for that."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'pic': True}
+
+source_urls = ["http://downloads.ghostscript.com/public/"]
+sources = [SOURCELOWER_TAR_GZ]
+
+# expat, freetype and libpng are optional deps,
+# these are actually patched and included in the sources,
```

Comment:
a recent easyconfig does include the deps here, cfr. https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/g/Ghostscript/Ghostscript-9.14-intel-2014b.eb

So it seems like that's a viable option?